cmd/ocsp-updater/main.go

```diff
@@ -422,6 +440,13 @@ func (updater *OCSPUpdater) oldOCSPResponsesTick(ctx context.Context, batchSize
 		return err
 	}
 
+	for _, s := range statuses {
```

Comment:
This seems to be in the wrong loop as it would mark any certificate that has an expired OCSP response as being expired itself which is incorrect.